plan/logical_plan_test.go

```diff
@@ -800,7 +804,7 @@ func (s *testPlanSuite) TestRefine(c *C) {
 		},
 		{
 			sql:  "select b from t where c = 1 or c = 2 or c = 3 or c = 4 or c = 5",
-			best: "Index(t.c_d_e)[[1,1] [2,2] [3,3] [4,4] [5,5]]->Projection",
+			best: "Table(t)->Projection",
```

Comment:
Because all the test use pseudo table, for a condition like c = 1 if we use the index c_d_e, it will be a range search in (1,-inf,-inf) to (1,+inf,+inf), since the `pseudoBetweenRate` is 4, so if we have more than four conditions like this, it is better to use a table scan.